packages/teleport/src/Apps/Empty/Empty.tsx

```diff
@@ -53,7 +53,14 @@ export default function Empty(props: Props) {
   }
 
   return (
-    <Card maxWidth="700px" mx="auto" py={4} as={Flex} alignItems="center">
+    <Card
+      maxWidth="700px"
+      mx="auto"
+      py={4}
+      as={Flex}
+      alignItems="center"
+      flex="0 0 auto"
```

Comment:
`flex="0 0 auto"` fixes this squishing flex box behavior in safari

![image](https://user-images.githubusercontent.com/43280172/108323278-52437200-717b-11eb-941d-279e83d5dae0.png)